docs/Xamarin.Forms.Core/Xamarin.Forms/IButtonController.xml

```diff
@@ -31,5 +31,37 @@
         <remarks>To be added.</remarks>
       </Docs>
     </Member>
+    <Member MemberName="SendPressed">
+      <MemberSignature Language="C#" Value="public void SendPressed ();" />
```

Comment:
No.  These are auto-generated by monodoc.  See also other APIS too. You can easily find many spaces like SendClicked ().  :-)